services/tasks/LocalJob_test.go

```diff
@@ -0,0 +1,56 @@
+package tasks
+
+import (
+	"testing"
+)
+
+func TestIsCLIArgsOverridden(t *testing.T) {
+	var args []struct {
+		tmplArg string
+		taskArg string
+		want    bool
+		err     error
+	} = []struct {
+		tmplArg string
+		taskArg string
+		want    bool
+		err     error
+	}{
+		{
+			tmplArg: "--ssh-extra-args=\"-p 3222\"",
+			taskArg: "--ssh-extra-args \"-p 3222\"",
```

Comment:
Single argument can not contain space which separates key and value. It must be 2 separate arguments. Otherwise `ansible-playbook` doesn't accept this argument (tested).

Reply:
I didn't get full what you mean? Do you mean that `--ssh-extra-args` will only work if separator is `=` sign? Like: `-ssh-extra-args="-p 3222"`? and won't work with `--ssh-extra-args "-p 3222"? If that's the case, how do you prevent this on template CLI args declaration already? And this is not a user mistake if that's the case? On the other hand this piece of code only is to determine if they are passing same argument from task CLI args, so that it can be overriden, I don't really touch CLI args how they look like, regex is just checking either first `=` or first white space and splits it from there, so it can compare the passed option, `--ssh-extra-args` in this case.

Reply:
@Gnyblast yes, it is true. Argument `--ssh-extra-args "-p 3222"` doesn't work in Semaphore.

You can try to call ansible CLI with params `ansible-playbook '--ssh-extra-args "-p 3222"'` and you will got error.

Semaphore runs ansible by command `exec.Command("ansible-playbook", []string{"--ssh-extra-args \"-p 3222\""})` so it will not work.